lib/rubocop/cop/rspec/let_setup.rb

```diff
@@ -3,7 +3,7 @@
 module RuboCop
   module Cop
     module RSpec
-      # Detect unreferenced `let!` calls being used for test setup
+      # Checks for `let!` being used for test setup.
```

Comment:
I think the previous message was better. `let!` is ok for test setup as long as the name is referenced.